qds_sdk/cluster.py

```diff
@@ -24,7 +24,7 @@ class Cluster(Resource):
     """
 
     rest_entity_path = "clusters"
-    api_version = "v1.2"
+    api_version = "v1.3"
```

Comment:
@anumahadevan this will cause the builds of our CI to also use the unreleased branch. And since the tests currently run from master branch this could potentially cause problems? Since the test factory work is still unmerged.